Packs/CommonScripts/TestPlaybooks/IsEmailAddressInternal_Test.yml

```diff
@@ -130,7 +131,7 @@ tasks:
       key:
         simple: emails
       value:
-        simple: '["email@domain.com", "email@sub.test.com", "invalidmail"]'
```

Comment:
You deleted the "invalidmail" email from the context values, as a result the task "IsEmailAddressInternal include_subdomains" with the email "invalidmail" won't be checked in your new test playbook. 
please add the same test with the error path to the "IsEmailAddressInternal include_subdomains" task.

Reply:
It was never checked.